packages/graphql-server/src/directives/makeDirectives.ts

```diff
@@ -50,9 +50,9 @@ export const makeDirectivesForPlugin = (
 export const getDirectiveName = (schema: DocumentNode) => {
   const definition = schema.definitions.find(
     (definition) => definition.kind === 'DirectiveDefinition',
-  ) as ExecutableDefinitionNode
+  ) as ExecutableDefinitionNode | undefined
```

Comment:
Looks like this cast isn't needed at all anymore
```suggestion
  )
```

Reply:
I'll get rid of it then. Thanks!